index.js

```diff
@@ -28,3 +29,20 @@ async function fetchCharacters() {
 }
 
 fetchCharacters();
+
+nextButton.addEventListener("click", () => {
+  if (page <= 42) {
+    page += 1;
+    fetchCharacters();
+    pagination.textContent = `${page} / 42`;
+    //console.log("page: ", page);
+  }
+});
+
+prevButton.addEventListener("click", () => {
+  if (page > 1) {
+    page -= 1;
+    pagination.textContent = `${page} / 42`;
+    fetchCharacters();
+  }
+});
```

Comment:
Same as above - logic is correct but please use `maxPage` instead of a hardcoded value.